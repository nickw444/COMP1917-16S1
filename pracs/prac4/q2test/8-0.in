8
1036 122 -1173 1048 -1145 1124 330 474 
