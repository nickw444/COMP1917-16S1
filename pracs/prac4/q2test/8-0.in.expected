sum of array: 1816
